src/alembic_utils/experimental/_collect_instances.py

```diff
@@ -24,7 +24,7 @@ def walk_modules(module: ModuleType) -> Generator[ModuleType, None, None]:
         # ...
     """
     top_module = module
-    top_path = top_module.__path__[0]  # type: ignore
+    top_path = PosixPath(top_module.__path__[0])
```

Comment:
Why is this `PosixPath` instead of `Path`?

won't that block windows support?

Reply:
I'm not sure. I don't have a Windows machine to test it on. But from my debugging, base_path is already a PosixPath--or do you think that is platform-dependent?

Reply:
Looks like you're right @olirice. Updated to use `Path`.